_config/config.yml

```diff
@@ -0,0 +1,5 @@
+---
+name: silverstripe-redirectedurls
+---
+RedirectedURLHandler:
+  case_insensitive_matching: false
```

Comment:
if you set a `private static` on `RedirectedURLHandler` instead of adding this config file then the code will be a bit more self documenting